### PR TITLE
feat: max-overflow on Avatar.Group to be triggered by click or focus …

### PR DESCRIPTION
**Pull request by RiteshMakan**
…(#31924)

Based on this issue, https://github.com/ant-design/ant-design/issues/31924. Here is a proposal to allow 'click' or 'hover' to trigger the popover instead

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by RiteshMakan** [comment]
Hi, all checks have passed, where can I ask for an approval to be able to merge this in?

**Comment by xrkffgg** [comment]
> * 🚨 Please fill changelog in the PR
>   
>   * Write with a **developer-oriented perspective** and **narrative method**, without describing the details of the repair
>   * **Describing the problem and the impact on the developer**
>   * **describing the user-first site problem**, not your solution
>   * Refer: https://ant.design/changelog#4.9.0
> * 🚨 请填写 PR 中的 changelog
>   
>   * 请用**面向开发者的角度**和**叙述方式撰写**，不描述修复细节
>   * **描述问题和对开发者的影响**
>   * **描述用户第一现场的问题**，而非你的解决方式
>   * 参考：https://ant.design/changelog-cn#4.9.0

ref

**Comment by RiteshMakan** [comment]
> > * 🚨 Please fill changelog in the PR
> >   
> >   * Write with a **developer-oriented perspective** and **narrative method**, without describing the details of the repair
> >   * **Describing the problem and the impact on the developer**
> >   * **describing the user-first site problem**, not your solution
> >   * Refer: https://ant.design/changelog#4.9.0
> > * 🚨 请填写 PR 中的 changelog
> >   
> >   * 请用**面向开发者的角度**和**叙述方式撰写**，不描述修复细节
> >   * **描述问题和对开发者的影响**
> >   * **描述用户第一现场的问题**，而非你的解决方式
> >   * 参考：https://ant.design/changelog-cn#4.9.0
> 
> ref

Hi, I pushed an update to the Changelog. I hope my Google translate to Chinese is ok. I see that the bot has closed my issue

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ timeline: true
   - 🆕 Tree support `onScroll` in virtual scroll. [#474](https://github.com/react-component/tree/pull/474)
   - 🐞 Fix Tree drop outside not clear indicator. [#478](https://github.com/react-component/tree/pull/478)
 - 💄 Add `@checkbox-border-radius` less variable. [#31360](https://github.com/ant-design/ant-design/pull/31360) [@Gherciu](https://github.com/Gherciu)
-- 🐞 Avatar add `crossOrigin` property to resolve cross origin issue. [#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
+- 🐞 Avatar
+  - add `crossOrigin` property to resolve cross origin issue. [#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
```
Comment (xrkffgg) [inline CHANGELOG.en-US.md:58]: Keep it as it is here.

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -54,7 +54,9 @@ timeline: true
   - 🆕 Tree 虚拟滚动也支持 `onScroll` 事件。[#474](https://github.com/react-component/tree/pull/474)
   - 🐞 修复 Tree 拖拽到外部时没有清空指示器的问题。[#478](https://github.com/react-component/tree/pull/478)
 - 💄 新增 Less 变量 `@checkbox-border-radius`。[#31360](https://github.com/ant-design/ant-design/pull/31360) [@Gherciu](https://github.com/Gherciu)
-- 🐞 Avatar 增加 `crossOrigin` 参数以解决跨域问题。[#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
+- 🐞 Avatar
```
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:57]: Keep it as it is here.

**Comment by xrkffgg** [comment]
Please input PR changelog

File: CHANGELOG.en-US.md (hunk 1 above)
Comment (RiteshMakan) [reply CHANGELOG.en-US.md:58]: 👍 Thank you. I want to take the time to thank you for assisting me with this PR. I really appreciate it

Comment (RiteshMakan) [reply CHANGELOG.en-US.md:58]: > Please input PR changelog

I have no idea what to do, I would like to learn. This is my first PR to ant-design

**Comment by RiteshMakan** [comment]
> Please input PR changelog

I have no idea what to do, I would like to learn. This is my first PR to ant-design

**Comment by xrkffgg** [comment]
image

open this

**Comment by RiteshMakan** [comment]
Screenshot 2021-09-03 at 10 32 06
Done

**Comment by xrkffgg** [comment]
There is some confusion, please open a new PR

**Pull request closed without merging**